ding_periodical.module

```diff
@@ -135,12 +136,17 @@ function theme_ding_periodical_issues($variables) {
 
       // Check if the volume is reservable and add reservation button.
       if (!empty($issue_id) && ding_periodical_is_reservable($issues[$key])) {
-        $path = array($issue_id, $volume, $key);
+        $item_id = array($issue_id, $volume, $key);
+
+        foreach ($item_id as &$v) {
+          $v = rawurlencode($v);
```

Comment:
The task was to get things work. There might be issues with the provided as was stated, yet it is the last concern to blame remote servers from our side.
As soon as the provider is fixed (which I'm not aware when this could be) this ofc should be brought back.